easy-ui-react/src/Tabs/Tabs.module.scss

```diff
@@ -68,12 +68,14 @@
   left: 0;
   bottom: 0;
   height: component-token("tab-nav", "indicator-height");
+  width: 1px;
 
   background: component-token("tab-nav", "current-page-color");
-  transform: translateX(#{component-token("tab-nav", "indicator-position")});
-  width: component-token("tab-nav", "indicator-width");
+  transform: translateX(#{component-token("tab-nav", "indicator-position")})
+    scaleX(#{component-token("tab-nav", "indicator-width")});
+  transform-origin: left;
 
-  transition-property: transform, width;
-  transition-duration: 0.2s;
+  transition-property: transform;
+  transition-duration: 0.25s;
```

Comment:
**question** how did you decide on this as a duration? is it something from spectrum?

Reply:
i didn't see it in spectrum. it was just based on what felt right and similar to what i've seen in other libraries in the past but i'm not married to it